package.json

```diff
@@ -121,5 +121,8 @@
     "tmp": "^0.0.33",
     "ws": "^6.1.2",
     "yargs": "^12.0.5"
+  },
+  "resolutions": {
+    "dir-glob": "2.0.0"
```

Comment:
By downgrading `dir-glob` from v2.2.0 to v2.0.0, failed tests caused by globby are fixed.

Perhaps the updated dir-glob cannot handle passed absolute path.

https://github.com/kevva/dir-glob/blob/3cd414fc3694dbaf9f87892febadf5703a6475ea/index.js#L6